packages/realm/README.md

```diff
@@ -1,24 +1,27 @@
-<picture>
-    <source srcset="https://github.com/realm/realm-js/blob/HEAD/media/logo-dark.svg" media="(prefers-color-scheme: dark)" alt="realm by MongoDB">
-    <img src="https://github.com/realm/realm-js/blob/HEAD/media/logo.svg" alt="realm by MongoDB">
-</picture>
+> [!NOTE]
+> Realm is now Atlas Device SDK - [Learn More](https://www.mongodb.com/blog/post/realm-now-part-atlas-platform?utm_medium=github_atlas_CTA&utm_source=realm_js_github)
 
-# About Realm Database
+<img align="right" style="min-width:150px;width:30%;" src="https://raw.githubusercontent.com/realm/realm-js/main/media/atlas-device-sync.svg" alt="Atlas Device Sync">
 
-Realm is a mobile database that runs directly inside phones, tablets or wearables.
-This project hosts the JavaScript versions of [Realm](https://www.mongodb.com/docs/realm/). Currently, we support React Native (JSC & Hermes on iOS & Android), Node.js and Electron (on Windows, MacOS and Linux).
+# Atlas Device SDK
+
+Atlas Device SDK is a suite of app development tools optimized for data access and persistence on mobile and edge devices. Use the SDKs to build data-driven mobile, edge, web, desktop, and IoT apps.
```

Comment:
What docs has been told and what we're working with is that Realm is the device data persistence layer part of Atlas Device SDK. Check out the Introduction page in the docs: https://www.mongodb.com/docs/realm/introduction/